--- /home/fonso/repos/york/modiff/org.eclipse.epsilon.modiff.demo/scenarios/swapjob/swapjob-from.model	2023-09-29 21:32:51.915998378 +0200
+++ /home/fonso/repos/york/modiff/org.eclipse.epsilon.modiff.demo/scenarios/swapjob/swapjob-to.model	2023-09-29 21:32:51.915998378 +0200
@@ -6,17 +6,18 @@
   <workers name="Alice"
       mainSkill="Disassembly"
       secondarySkills="Microsoldering DataRecovery">
+    <queue jobId="job2"
+        description="broken phone screen"/>
+  </workers>
+  <workers name="Bob"
+      mainSkill="Microsoldering">
     <queue jobId="job1"
         description="laptop does not charge">
       <status
           statusId="st1"
           status="pending"/>
     </queue>
-    <queue jobId="job2"
-        description="broken phone screen"/>
   </workers>
-  <workers name="Bob"
-      mainSkill="Microsoldering"/>
   <skills name="Microsoldering"/>
   <skills name="Disassembly"/>
   <skills name="DataRecovery"/>
