--- /home/fonso/repos/york/modiff/org.eclipse.epsilon.modiff.demo/scenarios/s01/s01-from.model	2023-09-29 21:32:51.915998378 +0200
+++ /home/fonso/repos/york/modiff/org.eclipse.epsilon.modiff.demo/scenarios/s01/s01-to.model	2023-09-29 21:32:51.915998378 +0200
@@ -7,7 +7,7 @@
       mainSkill="Disassembly"
       secondarySkills="Microsoldering DataRecovery">
     <queue jobId="job1"
-        description="laptop does not charge">
+        description="battery connector broken">
       <tags>battery</tags>
       <tags>charger</tags>
       <status
