--- base.model	2022-11-03 14:29:13.954933049 +0100
+++ left.model	2022-11-03 14:29:16.906856054 +0100
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ASCII"?>
 <comicshop:Shop xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:comicshop="comicshop" xmi:id="_rqtZkBUPEeut3qj6TjgmMQ" name="Travelling Man">
   <comics xmi:id="comic1" title="A Basketful of Heads"/>
-  <comics xmi:id="comic2" title="Cruel Summer"/>
+  <comics xmi:id="comic2" title="Bad Winter"/>
 </comicshop:Shop>
